pyracing/response_objects/session_data.py

```diff
@@ -19,21 +19,21 @@ def __init__(self, dict):
         self.laps_for_qual_avg = dict['nlapsforqualavg']
         self.laps_for_solo_avg = dict['nlapsforsoloavg']
         self.lead_changes = dict['nleadchanges']
-        self.league_id = dict['leagueid']
-        self.league_season_id = dict['league_season_id']
+        self.league_id = parse_encode(dict['leagueid'])
+        self.league_season_id = parse_encode(dict['league_season_id'])
         self.leave_marbles = dict['leavemarbles']
         self.max_weeks = dict['maxweeks']
-        self.points_type = dict['pointstype']
+        self.points_type = parse_encode(dict['pointstype'])
         self.private_session_id = dict['privatesessionid']
         self.race_week = dict['race_week_num']
-        self.reserve_status = dict['rserv_status']
+        self.reserve_status = parse_encode(['rserv_status'])
```

Comment:
you accidentally deleted the `dict` here

Reply:
Added back the ```dict``` to reserve_status